### ISSUE TITLE
Allow us to ignore "No internet connection" popup

### ISSUE THREAD

**Issue by ConEstArthur** [Arturo254/InnerTune#129]
I remember I was on 0.9.8 release I think and I could play songs that are cached on my phone without an internet connection. It worked so well.

I wish the new version of the app wouldn't force us to go to downloads and instead let us the use in offline mode anyway. It would let me play my top 50 even without downloading them.

![Screenshot_20240923-111212](https://github.com/user-attachments/assets/9ac060cd-648a-4647-a275-cbb5d44f9376)

**Comment by Arturo254** [comment]
Offline mode in the new version (1.3.0) now redirects you to your library

**Comment by ConEstArthur** [comment]
> Offline mode in the new version (1.3.0) now redirects you to your library

Yes but there's no option to select from History which is a bummer.
